### PR TITLE
FireFly 2023 Q4 update

### PR DESCRIPTION
**Pull request by aznrayizzle**

### CODE REVIEW

**Review by petermetz — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-FireFly.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+- Sibos 2023: [Managing Digital Assets at Scale: Lessons Learned and What’s Next](https://www.kaleido.io/resources/managing-digital-assets-at-scale-lessons-learned-and-whats-next)
+- Kaleido & AWS event: [Unlocking the Potential of Asset Tokenization](https://www.kaleido.io/nyc-tokenization-event)
+- All Things Open: [An Introduction to Open Source Tools for Building in Web3](https://2023.allthingsopen.org/sessions/an-introduction-to-open-source-tools-for-building-in-web3/)
+- Hyperledger Member Webinar: [The Emerging Tech Stack for Digital Assets and Asset Tokenization](https://www.youtube.com/watch?v=heVWZZnz-j4)
+
+### Contributor activity updates
+
+The star activity for the FireFly repository has continued to grow from 390 end of last quarter to 430 end of this quarter. Discord activity remains consistent with ~45 DAU.
+
+![alt_text](images/HLFF_2023_Q3_image1.png "image_tooltip")
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q4-Hyperledger-FireFly.md]: Assuming this needs to be changed to a Q4 image.

**Review by tkuhrt — CHANGES_REQUESTED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-FireFly.md (hunk 1 above)
Comment (tkuhrt) [reply project-reports/2023/2023-Q4-Hyperledger-FireFly.md]: @aznrayizzle : Can you please have a look at this change?

**Review by mbrandenburger — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-FireFly.md (hunk 1 above)
Comment (tkuhrt) [reply project-reports/2023/2023-Q4-Hyperledger-FireFly.md]: @aznrayizzle : Can you please update the document based on my feedback?

**Review by VRamakrishna — APPROVED** [review]

**Review by swcurran — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-FireFly.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+![alt_text](images/HLFF_2023_Q4_1.png "image_tooltip")
+
+![alt_text](images/HLFF_2023_Q4_2.png "image_tooltip")
+
+![alt_text](images/HLFF_2023_Q4_3.png "image_tooltip")
+
+![alt_text](images/HLFF_2023_Q4_4.png "image_tooltip")
+
+![alt_text](images/HLFF_2023_Q4_5.png "image_tooltip")
```
Comment (arsulegai) [inline project-reports/2023/2023-Q4-Hyperledger-FireFly.md:111]: Do you have insights to measure maintainers activity across organizations similar to contributor activities?

**Review by arsulegai — APPROVED** [review]

File: project-reports/2023/2023-Q4-Hyperledger-FireFly.md (hunk 2 above)
Comment (aznrayizzle) [reply project-reports/2023/2023-Q4-Hyperledger-FireFly.md:111]: The list of maintainers is here: https://wiki.hyperledger.org/display/FIR/Maintainers
Currently 10 from Kaleido, 1 from Fidelity, and 1 from Instinct Tools.

File: project-reports/2023/2023-Q4-Hyperledger-FireFly.md (hunk 1 above)
Comment (aznrayizzle) [reply project-reports/2023/2023-Q4-Hyperledger-FireFly.md]: @tkuhrt updated the image. Sorry about the delay in doing so.

Comment (tkuhrt) [inline project-reports/2023/2023-Q4-Hyperledger-FireFly.md]:
```suggestion
![alt_text](images/HLFF_2023_Q4_stars.png "image_tooltip")
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Please update the link in the report to match the new image name.

**Comment by jimthematrix** [comment]
thanks @tkuhrt for catching the image link mismatch. Just merged your suggestion!

**Review by tkuhrt — APPROVED** [review]

**Pull request merged**